static/js/services/contact-view-model-generator.js

```diff
@@ -0,0 +1,218 @@
+var _ = require('underscore');
+
+angular.module('inboxServices').factory('ContactViewModelGenerator',
+  function(
+    $log,
+    $q,
+    ContactSchema,
+    DB,
+    LineageModelGenerator,
+    Search
+  ) {
+    'ngInject';
+    'use strict';
+
+    var NAME_COMPARATOR = function(lhs, rhs) {
```

Comment:
I totes know the style guide brew

I mean, this is a function. It's not like we're editing other functions either. We could be using capital letters on all functions at this rate. 
I kindof don't really care at all though, and you probably have enough code review comments as it is. (and this is one more!)